lib/SensorFusion/src/App.h

```diff
@@ -131,18 +99,6 @@ class App
      */
     void fatalErrorHandler();
 
-    /**
-     * Callback for Command MQTT Topic.
-     * @param[in] payload   Payload of the MQTT message.
-     */
-    void cmdTopicCallback(const String& payload);
-
-    /**
-     * Callback for Motor Speeds MQTT Topic.
-     * @param[in] payload   Payload of the MQTT message.
-     */
-    void motorSpeedsTopicCallback(const String& payload);
```

Comment:
No motor control required anymore?

Reply:
No, this application should not include Remote Control.